stdlib/public/core/Pointer.swift

```diff
@@ -336,6 +336,7 @@ extension Int {
   ///
   /// - Parameter pointer: The pointer to use as the source for the new
   ///   integer.
+  @_alwaysEmitIntoClient
```

Comment:
Why this change? There was a symbol emitted from the stdlib and now there isn't 🤔 Although, it was `@_transparent` so the likelihood of anything actually linking this symbol is slim to none. 🤷

Reply:
D'oh, this slipped in from an earlier version that (rightly) assumed that generics over protocols will not implicitly opt into noncopyable constraints